test/unit/typescript/api-generation-test.ts

```diff
@@ -3,17 +3,15 @@ import {
   concurrent,
   errors,
   datastax,
-  geometry,
   mapping,
   metadata,
   metrics,
   policies,
   tracker,
   types,
-} from "../../../index";
-import * as root from "../../../index";
+} from "../../../main";
+import * as root from "../../../main";
```

Comment:
Did this test pass before this change, even though our `index` only contains Rust exports??

Reply:
Probably not, as TS was throwing errors from VScode. It's likely they are not being run at all.